101-sql-vm-new-storage/azuredeploy.json

```diff
@@ -0,0 +1,315 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "String",
+      "metadata": {
+        "description": "The name of the VM"
+      }
+    },
+    "virtualMachineSize": {
+      "type": "String",
+      "defaultValue": "Standard_DS13_v2",
+      "allowedValues": [
+        "Standard_DS13_v2",
+        "Standard_DS3_v2"
+      ],
+      "metadata": {
+        "description": "The virtual machine size."
+      }
+    },
+    "exisitingVirtualNetworkName": {
```

Comment:
need to include a resourceGroup name for the vnet as well and use it in the resourceIds

Reply:
added